autoload/nnn.vim

```diff
@@ -342,7 +344,14 @@ function! nnn#pick(...) abort
     let l:default_opts = { 'edit': 'edit' }
     let l:opts = extend(l:default_opts, get(a:, 2, {}))
     let s:temp_file = tempname()
-    let l:cmd = g:nnn#command.' -p '.shellescape(s:temp_file).' '.(l:directory != '' ? shellescape(l:directory): '')
+    if g:nnn#session ==# 'none'
+        let l:sess_cfg = ' '
+    elseif g:nnn#session ==# 'global'
+        let l:sess_cfg = ' -S '
+    elseif g:nnn#session ==# 'local'
```

Comment:
I don't see where you are removing this single instance session.

Reply:
Oh that is a good point. Since `nnn` does not expose an interface to delete sessions that will involve me re-writing all the configuration directory evaluation code though. Would this be better put in `nnn` itself?

Reply:
Never mind. That code already exists for finding and deleting the picker file. I will extract that out into a function for reuse.

Reply:
The reason is sessions are generally meant to be stored from a file manager point of view. What we can do is - remove the session file in picker mode if persistent session option is not used.

Also, now I see why you mentioned the pop-up. The user in this case doesn't know how the session is used.

I'll make both the changes in `nnn` now.